server/package.json

```diff
@@ -12,8 +12,8 @@
     "test": "cross-env NODE_ENV=test ./node_modules/mocha/bin/mocha --recursive ./test/bootstrap.test.js \"./test/**/*.test.js\" --exit",
     "test-service": "cross-env SQLITE_FILE_PATH=/tmp/gladys-test.db NODE_ENV=test ./node_modules/mocha/bin/mocha --recursive ./test/bootstrap.test.js \"./test/services/$npm_config_service/*.test.js\" \"./test/services/$npm_config_service/**/*.test.js\" --exit",
     "coverage": "nyc npm test && nyc report --reporter=text-lcov > coverage.lcov",
-    "prettier-check": "prettier --check '**/*.js' '**/*.json'",
-    "prettier": "prettier --write '**/*.js' '**/*.json'",
+    "prettier-check": "prettier --check \"**/*.js\" \"**/*.json\"",
+    "prettier": "prettier --write \"**/*.js\" \"**/*.json\"",
```

Comment:
Why did you change this? It doesn't seem related to the PR

Reply:
To make it work on Windows

Reply:
👍 Ok I need to check that it still works on other platform too then